src/common/ossl_helpers.c

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: MIT
+#include <oqs/oqs.h>
+#if defined(OQS_USE_OPENSSL)
+#include <openssl/evp.h>
+#include "ossl_helpers.h"
+#if OPENSSL_VERSION_NUMBER >= 0x30000000L
+static EVP_MD *sha256_ptr, *sha384_ptr, *sha512_ptr,
```

Comment:
Would it be sensible to initialize them all to NULL....

Reply:
`static` keyword does it for us